src/components/ShareModal/index.tsx

```diff
@@ -162,7 +166,7 @@ export default function ShareModal({
           </ButtonWithHoverEffect>
         </Flex>
         <InputWrapper>
-          <input type="text" value={shareUrl} />
+          <input type="text" value={shareUrl} onChange={noop} />
```

Comment:
This is to dismiss warning about setting value without setting onChange.
readOnly can be used instead, but it will prevent users from moving cursor. So setting onChange to noop is the better option